interoptest/src/pythonservice/httpserver.py

```diff
@@ -0,0 +1,83 @@
+# Copyright 2018 Google LLC
```

Comment:
will you have Http service running using two framwork (inbuilt python http and flask)?

Reply:
No, I'm using flask since we don't have appropriate lower-level instrumentation for `HttpServer`, but this means we're also testing the flask instrumentation here. I can remove the bare httpserver from this PR, but we should consider switching away from flask when we can.